ledger/byron/genesis_test.go

```diff
@@ -291,10 +291,11 @@ func TestGenesisNonAvvmUtxos(t *testing.T) {
 			expectedTxId,
 		)
 	}
-	if tmpUtxo.Output.Address().String() != testAddr {
+	addr := tmpUtxo.Output.Address()
+	if (&addr).String() != testAddr {
```

Comment:
This seems awkward. If `Address` uses a pointer receiver for its functions, we should have `Address()` above also return a pointer